modules/apps/commerce/commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceTestUtil.java

```diff
@@ -508,7 +508,7 @@ public static CommerceTaxFixedRate addCommerceTaxFixedRate(
 
 		CommerceTaxMethod commerceTaxMethod =
 			CommerceTaxMethodLocalServiceUtil.addCommerceTaxMethod(
-				userId, groupId, RandomTestUtil.randomLocaleStringMap(),
+				groupId, userId, RandomTestUtil.randomLocaleStringMap(),
```

Comment:
mmm, does this compile? Because the true at the end is strange, the active param now is at the beginning